From 70acaa3f5305b72e32c4be59168b473ce4e4e474 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Mon, 3 Mar 2008 21:36:40 +0200
Subject: [PATCH] rsnapshot-program.pl: Remove rsnapshot.conf.default

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 rsnapshot-program.pl |    9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

--- a/rsnapshot-program.pl
+++ b/rsnapshot-program.pl
@@ -63,7 +63,7 @@ my $have_lchown = 0;
 $| = 1;
 
 # version of rsnapshot
-my $VERSION = '@VERSION@';
+my $VERSION = '1.4.2';
 
 # command or interval to execute (first cmd line arg)
 my $cmd;
@@ -6681,12 +6681,7 @@ B<-D> a firehose of diagnostic informati
 =head1 CONFIGURATION
 
 B</etc/rsnapshot.conf> is the default configuration file. All parameters
-in this file must be separated by tabs. B</etc/rsnapshot.conf.default>
-can be used as a reference.
-
-It is recommended that you copy B</etc/rsnapshot.conf.default> to
-B</etc/rsnapshot.conf>, and then modify B</etc/rsnapshot.conf> to suit
-your needs.
+in this file must be separated by tabs.
 
 Long lines may be split over several lines.  "Continuation" lines
 B<must> begin with a space or a tab character.  Continuation lines will
